apps/barista/src/layout/scroll-to-top/scroll-to-top.html

```diff
@@ -0,0 +1,15 @@
+<button
+  class="ba-scroll-top-button"
+  [class.ba-scroll-top-button-show]="_showScrollToTop"
+  (click)="_scrollToTop()"
+>
+  <svg
```

Comment:
Using dt-icon here is not an option?

Reply:
No, because there is no icon that we could use.